File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SCH3001U00GrdSCH3100Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
	public class SCH3001U00GrdSCH3100Info
	{
		private String _jundalTable;
		private String _jundalPart;
		private String _gumsaja;
		private String _reserDate;
		private String _reserYn;
		private String _dataRowState;

		public String JundalTable
		{
			get { return this._jundalTable; }
			set { this._jundalTable = value; }
		}

		public String JundalPart
		{
			get { return this._jundalPart; }
			set { this._jundalPart = value; }
		}

		public String Gumsaja
		{
			get { return this._gumsaja; }
			set { this._gumsaja = value; }
		}

		public String ReserDate
		{
			get { return this._reserDate; }
			set { this._reserDate = value; }
		}

		public String ReserYn
		{
			get { return this._reserYn; }
			set { this._reserYn = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public SCH3001U00GrdSCH3100Info() { }

		public SCH3001U00GrdSCH3100Info(String jundalTable, String jundalPart, String gumsaja, String reserDate, String reserYn, String dataRowState)
		{
			this._jundalTable = jundalTable;
			this._jundalPart = jundalPart;
			this._gumsaja = gumsaja;
			this._reserDate = reserDate;
			this._reserYn = reserYn;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SCH3001U00GrdSCH3100Info.java
package nta.med.data.model.ihis.schs;

import java.sql.Timestamp;

public class SCH3001U00GrdSCH3100Info {
	private String jundalTable;
    private String jundalPart;
    private String gumsaja;
    private Timestamp reserDate;
    private String reserYn;
	public SCH3001U00GrdSCH3100Info(String jundalTable, String jundalPart,
			String gumsaja, Timestamp reserDate, String reserYn) {
		super();
		this.jundalTable = jundalTable;
		this.jundalPart = jundalPart;
		this.gumsaja = gumsaja;
		this.reserDate = reserDate;
		this.reserYn = reserYn;
	}
	public String getJundalTable() {
		return jundalTable;
	}
	public void setJundalTable(String jundalTable) {
		this.jundalTable = jundalTable;
	}
	public String getJundalPart() {
		return jundalPart;
	}
	public void setJundalPart(String jundalPart) {
		this.jundalPart = jundalPart;
	}
	public String getGumsaja() {
		return gumsaja;
	}
	public void setGumsaja(String gumsaja) {
		this.gumsaja = gumsaja;
	}
	public Timestamp getReserDate() {
		return reserDate;
	}
	public void setReserDate(Timestamp reserDate) {
		this.reserDate = reserDate;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
}
